packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,1245 @@
+    defaultMessage: 'Loading products'
+}
+
+const LOADING_MESSAGE = {
+    id: 'shipping_multi_address.loading.message',
+    defaultMessage: 'Loading products...'
+}
+
+const ERROR_LABEL = {
+    id: 'shipping_multi_address.error.label',
+    defaultMessage: 'Error loading products'
+}
+
+const ERROR_MESSAGE = {
+    id: 'shipping_multi_address.error.message',
+    defaultMessage: 'Error loading products. Please try again.'
+}
+
+const MAIN_LABEL = {
+    id: 'shipping_multi_address.main.label',
+    defaultMessage: 'Multi-shipping address selection'
+}
+
+const PRODUCTS_REGION_LABEL = {
+    id: 'shipping_multi_address.products.region',
+    defaultMessage: 'Products and delivery addresses'
+}
+
+const IMAGE_ALT = {
+    id: 'shipping_multi_address.image.alt',
+    defaultMessage: 'Product image for {productName}'
+}
+
+const DROPDOWN_LABEL = {
+    id: 'shipping_multi_address.dropdown.label',
+    defaultMessage: 'Delivery address options'
+}
+
+const ADDRESS_DESCRIPTION = {
+    id: 'shipping_multi_address.address.description',
+    defaultMessage: 'Address for {firstName} {lastName}'
+}
+
+const ADD_NEW_ADDRESS_DESCRIPTION = {
+    id: 'shipping_multi_address.add_new.description',
+    defaultMessage: 'Add a new delivery address'
+}
+
+const PRICE_LABEL = {
+    id: 'shipping_multi_address.price.label',
+    defaultMessage: 'Product price'
+}
+
+const ACTIONS_REGION_LABEL = {
+    id: 'shipping_multi_address.actions.region',
+    defaultMessage: 'Checkout actions'
+}
+
+const SUBMIT_DESCRIPTION = {
+    id: 'shipping_multi_address.submit.description',
+    defaultMessage: 'Continue to next step with selected delivery addresses'
+}
+
+const ADDRESS_LINE_2_FORMAT = {
+    id: 'shipping_multi_address.format.address_line_2',
+    defaultMessage: '{city}, {stateCode} {postalCode}'
+}
+
+const MultiShippingItemAttributes = ({variant, includeQuantity = true}) => {
+    // Get display values for attributes
+    const variationAttributes = variant?.variationAttributes || []
+    const variationValues = variant?.variationValues || {}
+    return (
+        <Stack spacing={1.5} flex={1} role="list" aria-label="Product attributes">
+            {variationAttributes &&
+                variationAttributes.length > 0 &&
+                variationAttributes.map((attr) => {
+                    const value = attr.values?.find((v) => v.value === variationValues[attr.id])
+                    return (
+                        <Text
+                            lineHeight={1}
+                            color="gray.700"
+                            fontSize="sm"
+                            key={attr.id}
+                            role="listitem"
+                        >
+                            {attr.name || attr.id}: {value?.name || value?.value || ''}
+                        </Text>
+                    )
+                })}
+            {includeQuantity && (
+                <Text lineHeight={1} color="gray.700" fontSize="sm" role="listitem">
```

Comment:
I saw you are setting the role here. How about we use chakra list component to achieve this? https://v2.chakra-ui.com/docs/components/list

Reply:
will refactor